core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -428,7 +428,9 @@ object GameStarter {
 
             //Trigger any global or nation uniques that should triggered.
             //We may need the starting location for some uniques, which is why we're doing it now
-            for (unique in ruleset.globalUniques.uniqueObjects + civ.nation.uniqueObjects)
+            val startingTriggers=(ruleset.globalUniques.uniqueObjects + civ.nation.uniqueObjects)
+                .filter { !it.conditionals.any { it.type!!.targetTypes.any { it.canAcceptUniqueTarget(UniqueTarget.TriggerCondition) } } }
```

Comment:
> !any == none?

Yes, though I didn't realize that none was an option there instead of !any

As for the explanation: I'm not sure the best way to add it as a comment. Maybe changing the names could help there, but a written explanation seems more appropriate in this situation as (imo) it's somewhat straightforwards once you're aware of how it functions. The trigger conditionals are supposed to be called when they come up, and the other conditionals are checked in `unique.conditionsApply`